bids/layout/layout.py

```diff
@@ -1316,3 +1319,4 @@ class Query(enum.Enum):
     """Enums for use with BIDSLayout.get()."""
     NONE = 1 # Entity must not be present
     ANY = 2  # Entity must be defined, but with an arbitrary value
+    ANY_OPTIONAL = 3  # Entity may or may not be defined
```

Comment:
`ANY_OPTIONAL`  feels clunky. What if we make `REQUIRED` a synonym for `ANY` and then use `OPTIONAL`?

```suggestion
    NONE = 1 # Entity must not be present
    ANY = REQUIRED = 2  # Entity must be defined, but with an arbitrary value
    OPTIONAL = 3  # Entity may or may not be defined
```

Or even make REQUIRED the default value, and let ANY be a backwards-compatible synonym:

```suggestion
    NONE = 1 # Entity must not be present
    REQUIRED = ANY = 2  # Entity must be defined, but with an arbitrary value
    OPTIONAL = 3  # Entity may or may not be defined
```

Reply:
I'm fine with `OPTIONAL`.

`REQUIRED` is cleaner, I feel like `ANY` is the misnomer here and maybe should be (eventually) deprecated.